src/App.jsx

```diff
@@ -1,10 +1,21 @@
 import "./App.css";
-import { Home } from "./pages";
+import { Navbar, Footer } from "./components/component";
+import { Home,WishList, ProductListing, Cart, Login, Signup } from "./pages";
+import { Routes, Route } from "react-router-dom";
 
 function App() {
   return (
     <div>
-      <Home />
+    <Navbar />
+      <Routes>
+        <Route path="/" element={<Home />} />
```

Comment:
you can put all these routes in another separate file.

Reply:
Ok I will do it